.github/workflows/cirrus_status.yml

```diff
@@ -92,7 +91,7 @@ jobs:
                 server_port: 465
                 username: ${{secrets.ACTION_MAIL_USERNAME}}
                 password: ${{secrets.ACTION_MAIL_PASSWORD}}
-                subject: Cirrus-CI cron build failures on ${{github.repository}}
+                subject: podman-machine-os build failures on ${{github.repository}}
```

Comment:
Kind of a nit but woudn't this end up saying "podman-machine-os build failures on containers/podman-machine-os"

Reply:
Fair enough, should I just say `Cirrus-CI build ...` then? Or maybe just `podman-machine-os build failures`?

Reply:
Well it is not always cron though, we build on ever push to main so post merge or cron.
That is why I removed the cron part.

Reply:
fixed